code/controllers/subsystems/ticker.dm

```diff
@@ -0,0 +1,560 @@
+						M.health = 0
+						M.set_stat(DEAD)
+				if(1)	//on a z-level 1 turf.
+					M.health = 0
+					M.set_stat(DEAD)
+
+	//Now animate the cinematic
+	switch(station_missed)
+		if(1)	//nuke was nearby but (mostly) missed
+			if( mode && !override )
+				override = mode.name
+			switch( override )
+				if("mercenary") //Nuke wasn't on station when it blew up
+					flick("intro_nuke",cinematic)
+					sleep(35)
+					world << sound('sound/effects/explosionfar.ogg')
+					flick("station_intact_fade_red",cinematic)
+					cinematic.icon_state = "summary_nukefail"
+				else
+					flick("intro_nuke",cinematic)
+					sleep(35)
+					world << sound('sound/effects/explosionfar.ogg')
+					//flick("end",cinematic)
+
+
+		if(2)	//nuke was nowhere nearby	//TODO: a really distant explosion animation
+			sleep(50)
+			world << sound('sound/effects/explosionfar.ogg')
+
+
+		else	//station was destroyed
+			if( mode && !override )
+				override = mode.name
+			switch( override )
+				if("mercenary") //Nuke Ops successfully bombed the station
+					flick("intro_nuke",cinematic)
+					sleep(35)
+					flick("station_explode_fade_red",cinematic)
+					world << sound('sound/effects/explosionfar.ogg')
+					cinematic.icon_state = "summary_nukewin"
+				if("AI malfunction") //Malf (screen,explosion,summary)
+					flick("intro_malf",cinematic)
+					sleep(76)
+					flick("station_explode_fade_red",cinematic)
+					world << sound('sound/effects/explosionfar.ogg')
+					cinematic.icon_state = "summary_malf"
+				if("blob") //Station nuked (nuke,explosion,summary)
+					flick("intro_nuke",cinematic)
+					sleep(35)
+					flick("station_explode_fade_red",cinematic)
+					world << sound('sound/effects/explosionfar.ogg')
+					cinematic.icon_state = "summary_selfdes"
+				else //Station nuked (nuke,explosion,summary)
+					flick("intro_nuke",cinematic)
+					sleep(35)
+					flick("station_explode_fade_red", cinematic)
+					world << sound('sound/effects/explosionfar.ogg')
+					cinematic.icon_state = "summary_selfdes"
+			for(var/mob/living/M in living_mob_list)
+				if(M.loc.z in using_map.station_levels)
+					M.death()//No mercy
+	//If its actually the end of the round, wait for it to end.
+	//Otherwise if its a verb it will continue on afterwards.
+	sleep(300)
+
+	if(cinematic)	qdel(cinematic)		//end the cinematic
+	if(temp_buckle)	qdel(temp_buckle)	//release everybody
+	return
+
+
+/datum/controller/subsystem/ticker/proc/create_characters()
+	for(var/mob/new_player/player in player_list)
+		if(player && player.ready && player.mind)
+			if(player.mind.assigned_role=="AI")
+				player.close_spawn_windows()
+				player.AIize()
+			else if(!player.mind.assigned_role)
+				continue
+			else
+				//VOREStation Edit Start
+				var/mob/living/carbon/human/new_char = player.create_character()
+				if(new_char)
+					qdel(player)
+				if(istype(new_char) && !(new_char.mind.assigned_role=="Cyborg"))
+					data_core.manifest_inject(new_char)
+				//VOREStation Edit End

```

Comment:
How is this a vorestation edit?

Reply:
No idea, this proc was not changed as part of this PR.  Only touch to these lines was to un-indent them by one level.